pages/solid-events.md

```diff
@@ -24,7 +24,10 @@ Solid Events provide an opportunity to meet others who are working on Solid or a
 
 | Date | Event | Organiser | Recordings, Slides and Notes |
 |----|-------|-------------|------------------------------|
-|2021-08-05|[Solid World](https://www.eventbrite.co.uk/e/solid-world-august-2021-tickets-158445927033?aff=erelexpmlt)|[Solid Team](https://solidproject.org/team)||
+|2021-09-02|[Solid World](https://www.eventbrite.co.uk/e/solid-world-september-2021-tickets-166560808863)|[Solid Team](https://solidproject.org/team)||
+|2021-08-05|[Solid World](https://www.eventbrite.co.uk/e/solid-world-august-2021-tickets-158445927033?aff=erelexpmlt)|[Solid Team](https://solidproject.org/team)| [SW Recording](https://vimeo.com/585006498) Ruben's Update: github.com/solid/community-server & 
+solidproject.org/self-hosting/css; Oz Update: [Compability Survey](research.net/r/solidappcompatibility); [Jeff’s Presentaion](github.com/jeff-zucker/deit/blob/main/Jeff-Talk.md); Kyra’s update: [DEI Team Gitter](gitter.im/solid/DEIT) Attend the DEI Team meeting – Tuesday, August 10, 11:00am ET
+[Meet on Jitsi](meet.jit.si/IntriguingSuspensionsSeekAhead), [Kyra's email](kyra.assaad@inrupt.com), [Twitter](Twitter.com/KyraAssaad); [Sarven’s Presentation](hackmd.io/RbcQL4SVQM6S17xVgYPyvA)|
```

Comment:
This doesn't quite look right, I think because the one table row spans multiple lines:

![image](https://user-images.githubusercontent.com/4251/128823610-3a85fdce-a522-445d-8426-e3f71d022cff.png)

(Seen by following the "details" link next to "netlify/solidproject-dot-org/deploy-preview" in this PR.)

Also, links need `https://` in front of them to work.

```suggestion
|2021-08-05|[Solid World](https://www.eventbrite.co.uk/e/solid-world-august-2021-tickets-158445927033?aff=erelexpmlt)|[Solid Team](https://solidproject.org/team)| [Recording](https://vimeo.com/585006498); Ruben's Update: https://github.com/solid/community-server & https://solidproject.org/self-hosting/css; Oz Update: [Compability Survey](https://research.net/r/solidappcompatibility); [Jeff’s presentation](https://github.com/jeff-zucker/deit/blob/main/Jeff-Talk.md); Kyra’s update: [DEI Team Gitter](gitter.im/solid/DEIT); Attend the DEI Team meeting – Tuesday, August 10, 11:00am ET [Meet on Jitsi](https://meet.jit.si/IntriguingSuspensionsSeekAhead), [Kyra's email](mailto:kyra.assaad@inrupt.com), [Twitter](https://Twitter.com/KyraAssaad); [Sarven’s Presentation](https://hackmd.io/RbcQL4SVQM6S17xVgYPyvA)|
```